objects/project-objects/scene.md

```diff
@@ -2,9 +2,16 @@
 
 ![](../../.gitbook/assets/iconscene.png)
 
-A **Scene** is a collection of **Objects** that represent different unique places within your user interface, such as menus, maps and media players. Each **Scene** has its on 3D space, with the **Size** and **Resolution** being determined by the **Attributes** of the **Screen** that it is assigned to.
+A **Scene** is a collection of Incari **Objects** such as **Mesh**, **Text**, **Sprite**, **Point Light**, **Spot Light**, etc. Each **Scene** can be rendered either in 2D or 3D space allowing for 2D or 3D views respectively. 
 
-The **Scene Object**, doesn't have any **Attributes** of its own, unlike every _other_ **Object** in **INCARI**.
+The 3D space is known as the **Editor view** in Incari and it allows a user to fully see how an **Object** functions in a **Scene**. Contrarily the 2D space mimics the view of Incari's **Player** and as such it is named the **Player view**. 
```

Comment:
Should be **Editor View** with both words capitalized. Same with Player View.

Reply:
Also, "Incari" itself should be bold. Just do a search and replace for all instances of `Incari` and replace with `**Incari**`.

You may need to then search for `****Incari****` and replace that with `**Incari**` as well if that makes sense! 😁